.github/workflows/michael-dev_taskemon-api2.yml

```diff
@@ -0,0 +1,67 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Backend - taskemon-api2
+
+on:
+  push:
+    branches:
+      - michael-dev

```

Comment:
Are we going to incorporate both main and develop here?

Reply:
I don't think its that simple unfortunately